symbolic-debuginfo/src/pdb.rs

```diff
@@ -117,8 +117,8 @@ impl From<pdb_addr2line::Error> for PdbError {
 ///
 /// This object is a sole debug companion to [`PeObject`](../pdb/struct.PdbObject.html).
 pub struct PdbObject<'data> {
-    pdb: Arc<RwLock<Pdb<'data>>>,
-    debug_info: Arc<pdb::DebugInformation<'data>>,
+    pdb: Rc<RefCell<Pdb<'data>>>,
+    debug_info: Rc<pdb::DebugInformation<'data>>,
```

Comment:
I would keep these as threadsafe `Arc` and `RwLock`, and just silence the clippy error.
See the comment below:

https://github.com/getsentry/symbolic/blob/89d6f326b028aa2492cef4fc2c9546caa0dfc708/symbolic-debuginfo/src/pdb.rs#L128-L133

Clippy is right that the inner type itself is not Send+Sync. However we explicitly make the outer type Send+Sync, so things will work out eventually.

Reply:
Even worse, the manual unsafe `Send+Sync` impl would be *really* wrong if we used non-threadsafe `Rc+RefCell` inside.

Reply:
Welp, should've read the comments 😅